app/services/description_import.rb

```diff
@@ -89,4 +95,36 @@ def compact_params(params)
       params[key].map { |param| compact_params(param) }
     end
   end
+
+  def remove_contributors_if_role_without_name(compacted_params_hash)
+    return unless compacted_params_hash && compacted_params_hash[:contributor]
+
+    compacted_params_hash[:contributor].each do |contributor|
+      next unless contributor && contributor[:name].nil? && contributor[:role].present?
+
+      compacted_params_hash[:contributor].delete(contributor)
+    end
```

Comment:
I *believe* Ruby already has an iterator that does this. Would you consider:

```suggestion
    compacted_params_hash[:contributor].delete_if do |contributor|
      contributor && contributor[:name].nil? && contributor[:role].present?
    end
```

❓

Reply:
great suggestion;  done.